src/chrome/content/rules/t66y.com.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="t66y.com">
+	<target host="t66y.com" />
+	<target host="www.t66y.com" />
+
```

Comment:
It prevents session cookie leaking from http hijack, can be important for users who have an account and never log out.